packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -720,3 +787,10 @@ class _OutputStreamContext implements _StreamContext {
 abstract interface class _StreamContext {
   StreamController get controller;
 }
+
+class _RequiredAuthentication {
```

Comment:
Question: Not sure exactly why, but somehow my brain doesn't expect `authInfo` here, because that is not "required" as such? The scopes are _required_, but the `authInfo` is just the current state. An example above in the code:
```dart
revokedAuthId.authId == requiredAuthentication.authInfo.authId,
```
In my head it would maybe make more sense with something like:
```dart
revokedAuthId.authId == currentAuthentication.authInfo.authId,
// or maybe even
revokedAuthId.authId == currentUser.authInfo.authId,
// or
revokedAuthId.authId == currentSessionAuth.authInfo.authId,
```

Maybe it's just me that has got the mental model wrong, let me know what your thoughts are!

Reply:
I understand the thinking, I think we could probably work on the naming and the class name.

What i want to assert is that we always evaluate the authentiction info used to establish the connection. Since the session lives for a longer time the we don't want this evaluation to change. If users want to use a different authentication info they will need to establish a new connection.

Reply:
[resolved offline] agreed on `AuthenticationContext` as new name